packages/isar/README.md

```diff
@@ -109,7 +109,10 @@ await isar.writeTxn((isar) {
 Isar has a powerful query language that allows you to make use of your indexes, filter distinct objects, use complex `and()` and `or()` groups and sort the results.
 
 ```dart
-final isar = await openIsar();
+final isar = await Isar.open(
+  schemas: [PostSchema],
+  path: await getDocuments,
```

Comment:
Randomly came across this PR and I think this should be a function call.
```suggestion
  path: await getDocuments(),
```

Good job with this lib, can't wait to use it in one of our apps 👍🏽